### PR TITLE
[BD-6] Add python 3.8 to the classifiers

### PR DESCRIPTION
**Pull request by morenol**
This already had tests in python 3.8 but now I am adding the classifiers

## Reviewers
- [x] @awais786 
- [x] @ericfab179 
- [x] Ready for edX review
- [ ] @jmbowman

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @morenol! I've created [OSPR-4437](https://openedx.atlassian.net/browse/OSPR-4437) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

File: .travis.yml [hunk 1]
```diff
@@ -2,13 +2,13 @@ language: python
...
-  - python: "3.5"
+  - python: 3.5
     env: DISTRIBUTIONS="sdist bdist_wheel --universal"
-  - python: "3.5"
+  - python: 3.5
     env: TOXARG="-e pylint"
-  - python: "3.6"
-  - python: "3.7"
-  - python: "3.8"
+  - python: 3.6
```
Comment (awais786) [inline .travis.yml:9]: Remove 3.6 and 3.7.

**Comment by awais786** [comment]
One minor comment otherwise seems ok.

File: setup.cfg [hunk 2]
```diff
@@ -13,8 +13,8 @@ classifiers =
     Intended Audience :: Developers
     License :: OSI Approved :: Apache Software License
     Operating System :: OS Independent
-    Programming Language :: Python :: 2.7
     Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.8
```
Comment (awais786) [inline setup.cfg:17]: @morenol remove this. Tests are not running on 3.8.

Comment (awais786) [reply setup.cfg:17]: You can just drop 2.7.
Add py3.8 with allowed failure in travis without py2.8 classifiers in this PR.
create new PR for py3.8 failures fix.

**Comment by edx-webhook** [comment]
@morenol Even though your pull request wasn’t merged, please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-lint&entry.867055334=https://github.com/edx/edx-lint/pull/115&entry.1484655318=https://github.com/morenol&entry.752974735=2020-04-30+16:35&entry.1917517419=2020-05-05+21:24&entry.2133058324=No) so we can improve your experience in the future.

File: setup.cfg (hunk 2 above)
Comment (morenol) [reply setup.cfg:17]: I think that the failure of the test was not related to this change https://travis-ci.org/github/edx/edx-lint/jobs/683180550, I triggered them again and they are ok

Comment (morenol) [reply setup.cfg:17]: Please review this again

**Comment by morenol** [comment]
@awais786 for a strange reason, github is not updating the checks from the last change, but they are green: https://travis-ci.org/github/edx/edx-lint/builds/683581046

**Comment by morenol** [comment]
@awais786 squashed, I think that this is ready to merge.

**Pull request merged**